internal/env.go

```diff
@@ -59,6 +60,22 @@ func DurationEnv(key string, def time.Duration) time.Duration {
 	return v
 }
 
+// IpEnv returns the valid IP string value of an environment variable, or
+// def otherwise.
+func IpEnv(key string, def string) string {
```

Comment:
nit: maybe we could use `net.IP` here instead of strings, WDYT?

Reply:
Changed in 22f3e0e 😄